ballerina/api_masterinspcharacteristic_srv/docs.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "The `ballerinax/sap.s4hana.api_masterinspcharacteristic_srv` package offers APIs for seamless integration with the [Master Inspection Characteristic - Read API v2.0.0](https://api.sap.com/api/API_MASTERINSPCHARACTERISTIC_SRV/overview).A master inspection characteristic is a master data record that describes the quality inspection criteria for materials. This service enables you to retrieve, for example, the settings and the control indicators defined for a master inspection characteristic.",
```

Comment:
```suggestion
  "description": "The `ballerinax/sap.s4hana.api_masterinspcharacteristic_srv` package offers APIs for seamless integration with the [Master Inspection Characteristic - Read API v2.0.0](https://api.sap.com/api/API_MASTERINSPCHARACTERISTIC_SRV/overview). This service allows to. read quality inspection specifications of the type master inspection characteristic. A master inspection characteristic is a master data record that describes the quality inspection criteria for materials.",
```

Reply:
Updated here [Update docs.json and gradle.properties](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/14/commits/b8c8fea8b5fc15e6761bed3bb7808449a7c7443d)